docs/buildah-manifest-push.1.md

```diff
@@ -18,6 +18,11 @@ The list image's ID and the digest of the image's manifest.
 
 ## OPTIONS
 
+**--add-compression** *compression*
+
+If specified while pushing option makes sure that requested compression variant exists for each platform in the manifest list.
```

Comment:
This sentence is a bit hard to brain parse.  I'd drop `while pushing` since we're already in the `buildah-push` man page.

We should also elaborate how in interacts with `--compression` and that the ones specified here are in _addition_ to the default or --compression one.

Reply:
Could you check if it looks any better now.